Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -175,12 +175,12 @@
   id: MobAghostGear
   equipment:
     back: ClothingBackpackSatchelHolding
-    id: AdminPDA
+    id: CMIDCardHighCommand # RMC-14: Changed from AdminPDA
```

Comment:
These changes will likely create conflicts on upsteam merge if upstream ends up changing them. I'd imagine it may be a good idea to inherit this prototype and use that instead of overriding base game one.

Reply:
I'm gonna leave this up to Smug or Whisper. The comments document the exact change made, if this causes a merge conflict it should be a no-brainer to fix.